modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro

```diff
@@ -898,6 +995,26 @@ definition {
 		return ${response};
 	}
 
+	macro relateObjectEntriesByName {
+		var childEntryId = ObjectDefinitionAPI.getObjectEntryIdByName(
+			en_US_plural_label = ${child_plural_label},
+			name = ${childEntryName});
+		var parentEntryId = ObjectDefinitionAPI.getObjectEntryIdByName(
+			en_US_plural_label = ${parent_plural_label},
+			name = ${parentEntryName});
+
+		var curl = ObjectDefinitionAPI._curlObjectEntriesRelationship(
+			en_US_plural_label = ${parent_plural_label},
+			objectEntry1 = ${parentEntryId},
+			objectEntry2 = ${childEntryId},
+			relationshipName = ${relationshipName},
+			virtualHost = ${virtualHost});
+
+		var response = JSONCurlUtil.put(${curl});
+
+		return ${response};
```

Comment:
Just wondering is this return necessary?

Reply:
FYI, in my opinion, if we use the return response when it has some errors we can debug it more easier